src/frequenz/client/dispatch/_client.py

```diff
@@ -221,7 +221,7 @@ async def create(
             active=active,
             dry_run=dry_run,
             payload=payload or {},
-            recurrence=recurrence or RecurrenceRule(),
+            recurrence=recurrence,
```

Comment:
Is this really necessary? If so could you explain why?

Reply:
The server (currently) expects a valid recurrence object, not a default-constructed one